src/main/java/com/launchdarkly/client/RedisFeatureStore.java

```diff
@@ -38,7 +48,10 @@
    * @param port the port for the Redis connection
    * @param prefix a namespace prefix for all keys stored in Redis
    * @param cacheTimeSecs an optional timeout for the in-memory cache. If set to 0, no in-memory caching will be performed
+   * @deprecated as of X. Please use the {@link com.launchdarkly.client.RedisFeatureStore#RedisFeatureStore(RedisFeatureStoreConfig)} constructor for a more
```

Comment:
Not sure what to set `X` to if we do decide deprecating is the way to go with these constructors